flood_adapt/object_model/scenario.py

```diff
@@ -1,19 +1,46 @@
-from pathlib import Path
+import os
+from typing import Any, Union
+
+import tomli
+import tomli_w
 
 from flood_adapt.object_model.direct_impacts import DirectImpacts
+from flood_adapt.object_model.hazard.hazard import ScenarioModel
+from flood_adapt.object_model.interface.scenarios import IScenario
 from flood_adapt.object_model.io.database_io import DatabaseIO
-from flood_adapt.object_model.site import SiteConfig
+from flood_adapt.object_model.site import Site
+
```

Comment:
Here would be the logical place for this, remove this from the hazard.py script. I added a comment there as well

```suggestion

class ScenarioModel(BaseModel):
    """BaseModel describing the expected variables and data types of a scenario"""
    name: str
    long_name: str
    event: str
    projection: str
    strategy: str
```

Reply:
see other comment